viewing_party/party.py

```diff
@@ -1,23 +1,239 @@
 # ------------- WAVE 1 --------------------
-
 def create_movie(title, genre, rating):
-    pass
+    """
+    Create a movie dictionary with title, genre, and rating.
+    Return None if any field is missing.
+    """
+    if not title or not genre or not rating:
+        return None
+    movie = {"title": title,
+            "genre": genre,
+            "rating": rating}
+    return movie

```

Comment:
Nice work on this function! Since it is relatively simplistic, checking for `Falsy` values and returning a dictionary, I would be okay with the suggested change below
```suggestion
    if not title or not genre or not rating:
        return None
    
    return {
        "title": title,
        "genre": genre,
        "rating": rating
    }
```

Remember that surrounding different blocks of logic in your function can help make your code more readable to yourself and other developers.